Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -2006,8 +2006,12 @@ AccessibilityObject* AccessibilityRenderObject::accessibilityHitTest(const IntPo
     if (!m_renderer || !m_renderer->hasLayer())
         return nullptr;
 
+    // Adjust point for the remoteFrameOffset
+    IntPoint& adjustedPoint = const_cast<IntPoint&>(point);
```

Comment:
This will modify the IntPoint in the scope of the caller.  Can we instead just copy point to a new IntPoint?